### PR TITLE
F fix/language & nutrients bugs

### PR DESCRIPTION
**Pull request by valeskus**

### CODE REVIEW

File: src/managers/LanguageManager/LanguageManager.ts [hunk 1]
```diff
@@ -19,8 +19,15 @@ export class LanguageManager {
 
    static async initLanguage() {
       const language = await this.getLanguage();
```
Comment (MAKARD) [inline src/managers/LanguageManager/LanguageManager.ts]: const language = await this.getLanguage() || 'en';

File: src/screens/RecipeDetails/hooks/useNutrientsController.ts [hunk 2]
```diff
@@ -80,14 +80,14 @@ export const useNutrientsController = (params: UseNutrientsControllerParams) =>
     const { numberOfServings } = params;
 
     return {
-      kCal: (recipe.kCal / recipe.servingsCount) * numberOfServings,
+      kCal: (recipe.kCal / recipe.servingsCount) * (numberOfServings || recipe.servingsCount),
```
Comment (MAKARD) [inline src/screens/RecipeDetails/hooks/useNutrientsController.ts]: should be fixed in another place

**Review by MAKARD — CHANGES_REQUESTED** [review]

File: src/screens/RecipeDetails/hooks/useServingsController.ts [hunk 3]
```diff
@@ -14,8 +14,12 @@ export const useServingsController = () => {
   }, [!!recipe]);
 
   const onCountChange = useCallback((value: number) => {
+    if (value === 0 && recipe) {
```
Comment (valeskus) [inline src/screens/RecipeDetails/hooks/useServingsController.ts]: the fix should not be here

File: src/UI/Counter/useCounterController.ts [hunk 4]
```diff
@@ -7,8 +7,8 @@ export interface UseCounterControllerParams {
 
 export const useCounterController = (params: UseCounterControllerParams) => {
   const onMinusPress = useCallback(() => {
-    if (params.count === 0) {
-      return;
+    if (params.count === 1) {
+      return params.onChange(params.count);
```
Comment (MAKARD) [inline src/UI/Counter/useCounterController.ts]: redundant call

File: src/screens/RecipeDetails/hooks/useServingsController.ts [hunk 5]
```diff
@@ -15,7 +15,7 @@ export const useServingsController = () => {
 
   const onCountChange = useCallback((value: number) => {
     setServingsCount(value);
-  }, []);
+  }, [recipe]);
```
Comment (MAKARD) [inline src/screens/RecipeDetails/hooks/useServingsController.ts]: redundant

**Review by MAKARD — CHANGES_REQUESTED** [review]

**Review by MAKARD — APPROVED** [review]

**Pull request merged**